examples/resources/auth0_role_permission/resource.tf

```diff
@@ -0,0 +1,37 @@
+# Example:
+resource "auth0_resource_server" "resource_server" {
```

Comment:
Let's have just a minimal configuration for this resource. The main protagonist of this example is the `auth0_role_permission` resource.